pkgs/top-level/all-packages.nix

```diff
@@ -26487,4 +26487,6 @@ in
   vpsfree-client = callPackage ../tools/virtualization/vpsfree-client {};
 
   gpio-utils = callPackage ../os-specific/linux/kernel/gpio-utils.nix { };
+
+  safety-cli = with python3.pkgs; toPythonApplication safety;
```

Comment:
`all-packages.nix` is in disarry, however, please attempt to place this in a more logical location